rotkehlchen/externalapis/bisq_market.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants.timing import DEFAULT_TIMEOUT_TUPLE
+from rotkehlchen.errors import RemoteError
+from rotkehlchen.history.deserialization import deserialize_price
+from rotkehlchen.typing import Price
+
+PRICE_API_URL = 'https://bisq.markets/api/ticker?market={symbol}_BTC'
+
+
+def get_bisq_market_price(asset: Asset) -> Price:
+    """
+    Get price for pair at bisq marketplace. Price is returned agains BTC.
+    Can raise:
+    - RemoteError: If the market doesn't exists or request fails
+    - DeserializationError: If the data returned is not a valid price
+    """
+    symbol = asset.symbol
+    url = PRICE_API_URL.format(symbol=symbol)
```

Comment:
Why declare a new variable just for this? How about?

```suggestion
    url = PRICE_API_URL.format(symbol=asset.symbol)
```